tests/manual/test-procedure.rst

```diff
@@ -446,6 +446,23 @@ Test Case: Autocomplete
 - [ ] Disable autocomplete in "Options" and confirm that autocomplete no longer offers suggestions.
 
 
+Test Case A11y: Keyboard focus follows a logical order
+''''''''''''''''''''''''''''''''''''''''''''''''''''''
+- [ ] Using your keyboard, navigate through the menu and any submenus using the `tab` and `enter` or `space` keys.
+- [ ] Using your keyboard, navigate through the modal windows using the `tab` and `enter` or `space` keys.
+
+
+Test Case A11y: Esc key returns focus to menu
+'''''''''''''''''''''''''''''''''''''''''''''
+- [ ] Type something un the text window, then confirm the `Esc` key returns the focus to the menu from the text window.
```

Comment:
Is the "text window" the "Script Name" text input box?

Reply:
The help calls the script/program window the Text window https://python-editor-2-0-0.microbit.org/help.html?snippets=true#text-window is there another term we use for it?

Reply:
Oh right, I guess I've been calling it the "text editor", but we don't really have any other source for a name.
"Window" to me means like a different OS app, or instance of the same app, but I guess "text editor" is not clear enough, as somebody could think of the entire Python Editor as a text editor. Is "text editor pane" more explicit? Not sure how common the word "pane" is outside of UI.

Reply:
"text editor" is probably more common. MakeCode use "Workspace", VICode uses "Editor". Should we change the term across the editor help and KB?

Reply:
Yeah, I think that would be beneficial, thanks Mark.

Reply:
Have used "text editor" in https://github.com/bbcmicrobit/PythonEditor/pull/304/commits/583c086845ced339529c69442d57a186387ff109